core/serialisation/data_serialiser.py

```diff
@@ -160,10 +160,10 @@ class Meta:
 
     submission_id = auto_field(model=Submission, data_key="SubmissionID")
     programme_id = auto_field(model=Programme, data_key="ProgrammeID")
-    question = fields.String(attribute="data_blob.question", data_key="Question")
-    indicator = fields.String(attribute="data_blob.indicator", data_key="Indicator")
-    response = fields.String(attribute="data_blob.response", data_key="Answer")
-    guidance_notes = fields.String(attribute="data_blob.guidance_notes", data_key="GuidanceNotes")
+    question = fields.String(attribute="data_blob.question", data_key="Question", dump_default=None)
```

Comment:
I wonder, it may be cleaner to try and make a base class that extends fields.String (and for the other fields methods we use) and always sets dump_default=None. Then in the serialisation classes we just invoke our custom class rather than fields.String? Means we don't have to repeat that kwarg every time, and also easier to change the behaviour accross the board by juts changing in one place?

Reply:
something like:

```
from marshmallow import fields

class CustomFloatField(fields.Float):
    def __init__(self, *args, **kwargs):
        kwargs.setdefault("dump_default", None)
        super().__init__(*args, **kwargs)
```

I haven't tried the above, it is from bing chat, so might be garbage :D

Reply:
I'll take a look at this

Reply:
Haven't thought about ```fields.Float``` or ```fields.Integer``` either. This needs to be QA'd as will effect data in extract